src/PkgTaskListWidget.cc

```diff
@@ -123,12 +120,13 @@ bool PkgTaskListWidgetItem::sortByInsertionSequence() const
     if ( ! listWidget() )
         return false;
 
-    PkgTaskListWidget * parentPkgTaskListWidget =
+    const PkgTaskListWidget * parentPkgTaskListWidget =
```

Comment:
Why?

This does a dynamic cast to a non-const pointer and then assigns that to a const pointer. This looks like a promise that the underlying object will not be modiefied, which is clearly not the case. This `const` only adds doubts and confusion.